drivers/display/ssd1327.c

```diff
@@ -12,19 +13,26 @@ LOG_MODULE_REGISTER(ssd1327, CONFIG_DISPLAY_LOG_LEVEL);
 #include <zephyr/init.h>
 #include <zephyr/drivers/display.h>
 #include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/i2c.h>
 #include <zephyr/drivers/mipi_dbi.h>
 #include <zephyr/kernel.h>
 
 #include "ssd1327_regs.h"
 
-#define SSD1327_ENABLE_VDD				0x01
-#define SSD1327_ENABLE_SECOND_PRECHARGE			0x62
-#define SSD1327_VCOMH_VOLTAGE				0x0f
-#define SSD1327_PHASES_VALUE				0xf1
-#define SSD1327_DEFAULT_PRECHARGE_V			0x08
-#define SSD1327_UNLOCK_COMMAND				0x12
+#define SSD1327_ENABLE_VDD         0x01
+#define SSD1327_UNLOCK_COMMAND     0x12
+#define SSD1327_MAXIMUM_CMD_LENGTH 16
+
+typedef int (*ssd13127_write_bus_cmd_fn)(const struct device *dev, const uint8_t cmd,
+					 const uint8_t *data, size_t len);
+typedef int (*ssd13127_write_pixels_fn)(const struct device *dev, const uint8_t *buf,
```

Comment:
typo 13127 --> 1327

Reply:
I thought I had fixed that, not sure what happened. Done.